src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -302,80 +298,6 @@ private void RaiseResumed()
 
             private static T NextOrDefault<T>(IEnumerator<T> enumerator)
                 => enumerator.MoveNext() ? enumerator.Current : default;
-
-            /// <summary>
-            /// Return all the spans that appear in only one of "latestSpans" or "previousSpans".
-            /// </summary>
-            private static DiffResult Difference<T>(IEnumerable<ITagSpan<T>> latestSpans, IEnumerable<ITagSpan<T>> previousSpans, IEqualityComparer<T> comparer)
-                where T : ITag
```

Comment:
this moved from a static/generic member to just a simple instance non-static member.  Ther was no need for generic as this is always instantiated with the outer TTag type that we already ahve access to.  It also is only called from one exact location, so even having it be shared and available for other callers is pointless.